compiler/compilerScript.sml

```diff
@@ -128,7 +128,7 @@ val error_to_str_def = Define`
        concat [strlit "### ERROR: type error\n"; s; strlit "\n"]
      else s) /\
   (error_to_str (ConfigError s) = concat [strlit "### ERROR: config error\n"; s; strlit "\n"]) /\
-  (error_to_str CompileError = strlit "### ERROR: compile error\n")`;
+  (error_to_str AssembleError = strlit "### ERROR: compile error\n")`;
```

Comment:
Can we change the error message too to say assembly or assemble instead of compile?

Reply:
Done!